### DISCUSSION TITLE
Difference between opinionated and unoppinionated packages

### DISCUSSION THREAD

**Discussion by katerlouis** [nuxt/eslint-config#256] [Q&A]
It just took over an hour to realize that there's a difference between `@nuxtjs/eslint-config` and `@nuxt/eslint-config` :F – yaix.

But still.. Why is one extend saying just `@nuxtjs` (without `eslint-config`) while the other is explicitly saying `@nuxt/eslint-config` – Is that eslint magic just assuming/searching for an `eslint-config` inside of `@nuxtjs` (nuxt**js**!!)? – If thats the case, I can tell you from a noob perspective, this is really confusing and I would expect the docs to treat both extends the same; either with or without `eslint-config`

What file is `@nuxtjs` using anyway? Is it `packages/eslint-config-legacy`?

And why exactly does `@nuxtjs` need babel-parser? For me its working without it (which is kinda scary to be honest :D)

Is there a recommendation for nuxt3 or do both the opinionated and unopinionated version work equally fine with nuxt3? It feels like `@nuxtjs` is nuxt2 and just `@nuxt` is nuxt3, like with nuxtjs.org and nuxt.com 🤷 

The opinionated version is giving me an error for undefined `onMounted()` while the unopinionated is cool with it; is that by design or are there just so few rules in the unopionated? 

Sorry for these questions, but eslint with its magic and nuxt/nuxtjs is really confusing to me :(

**Comment by riddla** [comment]
@katerlouis, I second your confusion about the almost identically named packages/configs. At least the (nonfunctional) `@nuxtjs` import has been corrected in the documentation via https://github.com/nuxt/eslint-config/commit/06b5c88a5ac9ebef5336c6bf6596a6d6871057f7. 

So the new config meant to be used in Nuxt 3 seems to be `@nuxt/eslint-config` and is currently in [version 0.1.1](https://www.npmjs.com/package/@nuxt/eslint-config/v/0.1.1). Unfortunately there seems to be not much progress since last december and the README also has a lot of (old) `@nuxtjs/eslint-config*` fragments.

**Comment by ColtHands** [comment]
I think rules that this config applies and differences between `@nuxtjs/eslint-config` and `@nuxt/eslint-config` should be documented.
